conf/config-default.yaml

```diff
@@ -359,7 +359,7 @@ plugins:                          # plugin list (sorted by priority)
   - limit-req                      # priority: 1001
   #- node-status                   # priority: 1000
   - gzip                           # priority: 995
-  - server-info                    # priority: 990
+  # - server-info                    # priority: 990
```

Comment:
Is this the result of discussion?

Reply:
No, I can't fix the test case failed, and I am looking for help.